src/assessment/builder.py

```diff
@@ -165,6 +173,11 @@ def build_campaigns(assessment, campaign_number, template_smtp):
         else:
             logging.error("Complete Date is not after Launch Date.")
 
+        if campaign.complete_date > datetime.now(campaign_tz).isoformat():
+            break
+        else:
+            logging.error("Complete date is not after the current datetime.")
+
```

Comment:
Is it worth adding a check that the campaign completion date is after the campaign start date?

Reply:
I have a check in there to ensure complete date is after the start date and after the current time

See here: 5a0cc25762521a630a682bdebea6964e12f4dbe9